addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12222,7 +12225,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#23050"
-msgid "Activate teletext"
+msgid "Activate Teletext"
```

Comment:
Zeroconf seems to be written uppercased in most resources I found. Also I think that it being a name of a protocol/service qualifies for uppercased.